docker-compose.yml

```diff
@@ -194,6 +194,8 @@ services:
       - ./localSetup/projects/content-author/startup.sh:/startup.sh
       - ./sourcecode/apis/contentauthor:/app
       - ./data/nginx/ca:/usr/local/share/ca-certificates:ro
+      - ./sourcecode/apis/contentauthor/vendor/h5p/h5p-editor:/app/public/h5p-editor-php-library
```

Comment:
I'd prefer if CA could check for the existence of these files in the correct directory instead.

Reply:
Check in vendor folder if H5P language file exists